src/constants.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * 싱수
+ */
+
+export const ERROR_MESSAGE = Object.freeze({
+  LOTTO_NUMBER_INPUT_ERROR:
+    "[ERROR] 로또 번호는 1부터 45 사이의 숫자여야 합니다.",
+  BONUS_NUMBER_INPUT_ERROR:
+    "[ERROR] 보너스 번호는 1부터 45 사이의 숫자여야 합니다.",
+  LOTTO_MONEY_INPUT_ERORR:
+    "[ERROR] 구입 금액은 1,000원 단위로 입력해야 합니다.",
+  LOTTO_MONEY_INPUT_STRING_ERORR:
+    "[ERROR] 구입 금액은 숫자 형태로 입력해야 합니다.",
+  WINNING_LOTTO_LENGTH_ERROR: "[ERROR] 로또 당첨 번호는 6개여야 합니다.",
+  WINNING_LOTTO_NUMBER_ERROR:
+    "[ERROR] 당첨 번호는 숫자 형태로 입력해야 합니다.",
+  WINNING_LOTTO_MAX_MIN_ERROR:
+    "[ERROR] 당첨 번호는 1~45 사이의 숫자여야 합니다.",
+  WINNING_LOTTO_NUMBER_DUPLICATE_ERROR:
+    "[ERROR] 당첨 번호가 중복이 됩니다. 1~45까지 중복이 안되도록 입력해주세요.",
+  WINNING_LOTTO_BONUS_NUMBER_ERROR:
+    "[ERROR] 보너스 번호는 숫자 형태로 입력해야 합니다.",
+  WINNING_LOTTO_BONUS_NUMBER_DUPLICATE_ERROR:
+    "[ERROR] 당첨 번호와 보너스번호가 중복됩니다.",
```

Comment:
[airbnb 23.10](https://github.com/airbnb/javascript?tab=readme-ov-file#naming--uppercase) 규칙 중에 상수를 선언할 때 top-level에서만 uppercase를 적용한다는 컨벤션이 있습니다!
그리고 변수 이름이 많이 길어진다면 case별로 묶어서 사용줘도 좋을 것 같아요

```suggestion
bonusNumber : {
    error:   "[ERROR] 보너스 번호는 숫자 형태로 입력해야 합니다.",
    duplicateError : "[ERROR] 당첨 번호와 보너스번호가 중복됩니다.",
}
```

과 같은 형식으로요!